src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java

```diff
@@ -142,6 +167,57 @@ public void sendNotifyEmails(CaseDetails caseDetails) {
         }
     }
 
+    public Optional<SendNotificationTypeItem> getSendNotification(CaseData caseData) {
+        return caseData.getSendNotificationCollection()
+                .stream()
+                .filter(s -> s.getId().equals(caseData.getSelectNotificationDropdown().getSelectedCode()))
+                .findFirst();
+    }
+
+    private String getSendNotificationSingleDocumentMarkdown(DocumentTypeItem documentTypeItem) {
+        String document = "| Document | N/A |";
+        if (documentTypeItem != null) {
+            Matcher matcher = Helper.getDocumentMatcher(
+                    documentTypeItem.getValue().getUploadedDocument().getDocumentBinaryUrl());
+            String documentLink = matcher.replaceFirst("");
+            String documentName = documentTypeItem.getValue().getUploadedDocument().getDocumentFilename();
+            document = String.format("| Document | <a href=\"/documents/%s\" target=\"_blank\">%s</a>| ", documentLink,
+                    documentName);
+        }
+        return document;
+    }
```

Comment:
There now exists a helper method for this in `Helper`. Would prefer using this instead of duplicating code